lib/dialogs/sell/usecases/GetAPTSellInfoUseCase.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:ax_dapp/dialogs/buy/usecases/GetSwapInfoUseCase.dart';
+import 'package:ax_dapp/service/BlockchainModels/AptSellInfo.dart';
+import 'package:ax_dapp/service/Controller/Swap/AXT.dart';
+import 'package:fpdart/fpdart.dart';
+
+const String _no_sell_info_error_msg = "No sell info found";
+
+class GetAPTSellInfoUseCase {
```

Comment:
Looks very similar to GetBuyInfoUseCase... Perhaps we can consolidate the two? Buy is a swap between the targetApt and Ax so maybe the GetBuyInfoUseCase should just call this with a hardcoded tokenFrom value